turbo.json

```diff
@@ -4,10 +4,7 @@
     "build": {
       "outputs": ["dist/**"]
     },
-    "test": {
-      "dependsOn": ["build"],
-      "inputs": ["src/**/*.tsx", "src/**/*.ts", "test/**/*.ts", "test/**/*.tsx"]
-    },
+    "test": {},
```

Comment:
`test` shouldn't have a topological dependency on `build`. Also the inputs aren't necessary AFAICT.